gradle-plugin/plugin/src/functionalTest/kotlin/com/emergetools/android/gradle/NoAppProjectPathPluginTest.kt

```diff
@@ -1,14 +0,0 @@
-package com.emergetools.android.gradle
-
-import com.emergetools.android.gradle.base.EmergeGradleRunner
-import org.junit.jupiter.api.Test
-
-class NoAppProjectPathPluginTest : EmergePluginTest() {
-  @Test
-  fun noAppProjectPath() {
-    EmergeGradleRunner.create("no-app-project-path")
```

Comment:
I could not figure out what this test was supposed to do. Please let me know if I should keep it.